src/Pages/Detail/Detail.js

```diff
@@ -1,13 +1,302 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import SpaceInfo from "./SpaceInfo";
+import DetailNav from "./DetailNav";
+import DetailList from "./DetailList";
+import Facility from "./Facility";
+import Tags from "./Tags";
+import Reservation from "./Reservation";
+import Refund from "./Refund";
+import Review from "./Review";
+import "./Detail.scss";
 
-class componentName extends Component {
+class Detail extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      space_id: 0,
+      header: "",
+      subDescription: "",
+      price: "",
+      time: 2,
+      num: 10,
+      imgUrl: [],
+      spaceDesc: "",
+      subHeader: "",
+      tag: [],
+      spaceIntro: "",
+      opening_hour: "",
+      closing_hour: "",
+      closed_day: "",
+      spaceType: [],
+      numValue: 1,
+      reservation: "",
+      amenities: [],
+      reviewNum: 2,
+      reviews: [],
+      selectDate: "",
+      selectStartTime: "",
+      selectEndTime: "",
+      totalNumber: "",
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://18.223.188.215:8000/spaces/detail/3")
```

Comment:
api 주소도 계속 바뀌고 detail 페이지 몇 번 받아오는 지도 바뀌니까 변수로 하셔야 해요!
```suggestion
    fetch(`${API}/spaces/detail/${여기도 변수}`)
```

Reply:
넵넵